Makefile

```diff
@@ -34,6 +34,10 @@ clean:
 test:
 	./hack/test integration gateway dockerfile
 
+.PHONY: test-race
+test-race:
```

Comment:
Maybe this new target is overkill, and it's sufficient to just indicate the new `GO_RACE_ENABLED=1` env in the CONTRIBUTING.md?